servicedirectory/snippets_test.py

```diff
@@ -34,11 +35,16 @@
 def teardown_module():
     client = servicedirectory_v1.RegistrationServiceClient()
     namespace_name = client.namespace_path(PROJECT_ID, LOCATION_ID, NAMESPACE_ID)
-    try:
-        namespace = client.get_namespace(name=namespace_name)
-        client.delete_namespace(name=namespace.name)
-    except exceptions.NotFound:
-        pass
+    all_namespaces = quickstart.list_namespaces(PROJECT_ID, LOCATION_ID).namespaces
+
+    # Delete namespace only if it exists
+    for namespace in all_namespaces:
+        if namespace_name in namespace.name:
+            try:
+                client.delete_namespace(name=namespace_name)
+            except exceptions.NotFound:
+                pass
```

Comment:
```suggestion
                print("Namespace already deleted")
```

Reply:
just a small nit - except/pass is a pet peeve of mine and I like to print something useful instead

Reply:
I agree! Done.